### DISCUSSION TITLE
Can I add this type of Formate?

### DISCUSSION THREAD

**Discussion by MuhammadEhsan2** [Serene-Arc/bulk-downloader-for-reddit#330] [Q&A] [answered]
@aliparlakci @Serene-Arc
{
    "name": "MyTopic",
    "subreddits": [ {"name":  "first-sub", "sort": "top", "time":  "week", "limit": 10},
                    {"name": "second-sub", "sort": "hot", "time": "month", "limit": 20} ],
    "MAX_CLIP_TIME":    60.0,  # seconds
    "MAX_SUBCLIP_TIME": 20.0,  # seconds
}

**Can I mention this type of arguments? And these arguments accept only one parameter at a time?**

time: accepts hour, day, week, month, year and all

1. year: 2018
2. month: 12
3. day: 24
4. time: 04:59:31

**Comment by Serene-Arc** [comment, marked as answer]
No, the time parameter in this case refers to the time filter on Reddit. You can only specify one of them and it will result in a listing of submission of that time or less. So if you use "day", it will only find submissions posted in the last day. You can't specify more than one and they don't take any arguments.

**Reply by MuhammadEhsan2** [reply]
If I want last month post (videos). So, which query am i using?

Query:
 "time": "month",

In this case, I download only this month post not previous one.

**Reply by Serene-Arc** [reply]
To my knowledge that isn't possible through the Reddit interface.

**Reply by MuhammadEhsan2** [reply]
okay Thank You !<3

**Reply by Serene-Arc** [reply]
As a rule of thumb, if you can do it through the normal Reddit website, there might be a way to do it in the BDFR. But afaik there's no way to do that on the website, which means that it's impossible through the python API.

**Comment by MuhammadEhsan2** [comment]
okay Thanks
